draft-ietf-rats-eat.md

```diff
@@ -685,33 +714,81 @@ more security-oriented subsystems like a TEE or a Secure Element.
 
 The claims for each these can be grouped together in a submodule.
 
-Specifically, the "submods" claim is an array. Each item in the array
-is a CBOR map containing all the claims for a particular submodule.
+The submods part of a token a single map/object with many entries, one
+per submodule.  There is only one submods map in a token. It is
+identified by its specific label. It is a peer to other claims, but it
+is not called a claim because it is a container for a claim set rather
+than an individual claim. This submods part of a token allows what
+might be called recursion. It allows claim sets inside of claim sets
+inside of claims sets...
+
+### Two Types of Submodules
+
+Each entry in the submod map one of two types:
+
+* A non-token submodule that is a map or object directly containing
+  claims for the submodule.
+* A nested EAT that is a fully-formed, independently signed EAT token
+
+#### Non-token Submodules
 
-The security level of the submod is assumed to be at the same level as
-the main entity unless there is a security level claim in that
-submodule indicating otherwise. The security level of a submodule can
-never be higher (more secure) than the security level of the EAT it is
-a part of.
+Essentially this type of submodule, is just a sub-map or sub-object
+containing claims. It is recognized from the other type by being
+a data item of type map in CBOR or by being an object in JSON.
 
-### The submod_name Claim
+The contents are claims about the submodule of types defined 
+in this document or anywhere else claims types are defined.
 
-Each submodule should have a submod_name claim that is descriptive
-name. This name should be the CBOR txt type.
+#### Nested EATs
+
+This type of submodule is a fully formed EAT as described here. In
+this case the submodule has key material distinct from the containing
+EAT token that allows it to sign on its own.
+
+When an EAT is nested in another EAT as a submodule the nested EAT
+MUST use the CBOR CWT tag. This clearly distinguishes it from the
+non-token submodules.
+
+### No Inheritance
+
+The subordinate modules do not inherit anything from the containing
+token.  The subordinate modules must explicitly include all of their
+claims. This is the case even for claims like the nonce and age.
+
+This rule is in place for simplicity. It avoids complex inheritance
+rules that might vary from one type of claim to another. (TODO: fix
+the boot claim which does have inheritance as currently described).
+
+### Security Levels and Attachment
+
+The security level of the non-token subordinate modules should always
```

Comment:
If submods are used structurally within an EAT, they should be able to have the same level of security as the EAT (rather than making lesser security mandatory). May just be unfortunate usage of the word 'should' and can be stated as 'not higher'?

Reply:
Oversight. Should be less than or equal too. Thanks for catching.